internal/cmd/controller/target/rollout.go

```diff
@@ -51,6 +57,10 @@ func manualPartition(rollout *fleet.RolloutStrategy, targets []*Target) ([]parti
 					continue targetLoop
 				}
 			}
+			if len(target.ClusterGroups) == 0 && matcher.Match(target.Cluster.Name, "", nil, target.Cluster.Labels) {
+				partitionTargets = append(partitionTargets, target)
+				continue targetLoop
+			}
```

Comment:
This addition fixes an issue where partitions name a single cluster, which would not result in a partition with the named cluster.

Reply:
Just noticed that we don't even have it documented. But if we decide to document it, it'll actually work.

Reply:
Is this covered by test case `should match cluster names` in `rollout_test.go`? That's the impression I get, but just to be sure.